test/doctool/test-doctool-html.js

```diff
@@ -89,14 +96,25 @@ testData.forEach(function(item) {
           filename: 'foo',
           template: 'doc/template.html',
           nodeVersion: process.version,
+          analytics: item.analyticsId,
         },
         common.mustCall((err, output) => {
           assert.ifError(err);
 
           const actual = output.replace(/\s/g, '');
           // Assert that the input stripped of all whitespace contains the
           // expected list
-          assert.notEqual(actual.indexOf(expected), -1);
+          assert.notStrictEqual(actual.indexOf(expected), -1);
+
+          // Testing the insertion of Google Analytics script when
+          // an analytics id is provided. Should not be present by default
+          if (includeAnalytics) {
+            assert.notStrictEqual(actual.indexOf('google-analytics.com'), -1,
+                                  'Google Analytics script was present');
```

Comment:
Not really, but I see the convention is opposite of what I'm used to so I just pushed an update -- thanks.